airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -213,11 +213,12 @@
             <property name="allowEmptyConstructors" value="true" />
             <property name="allowEmptyMethods" value="true" />
             <property name="allowEmptyLambdas" value="true" />
+            <property name="allowEmptyTypes" value="true" />
```

Comment:
> The `allowEmptyTypes` option needs to be added so that empty anonymous
> types do not cause violation because of missing space before `LCURLY`:
> ```
> doSomethingWith(Key.get(new TypeLiteral<MyType>() {}, AnnotatedWith.class)); // this is fine
> ```

Your example _does_ have a space before left left curly. What's the issue?

Reply:
It's "whitespace *around*", so it would also complain about missing space after the left curly.